src/main/java/Main.java

```diff
@@ -1,36 +1,17 @@
-
-import java.awt.event.ActionEvent;
-import java.util.ArrayList;
-
-import javax.swing.JFrame;
-
 import controllers.GameCreationController;
 import controllers.PlayerGameController;
 import database.CSVDatabase;
-import entities.*;
-import enums.Rank;
-import enums.Suit;
-import enums.WindowName;
-import ui.components.NavigationButton;
-import ui.factories.CreatorWindowFactory;
-import ui.factories.GameWindowFactory;
-import ui.factories.MenuWindowFactory;
-import ui.factories.RuleWindowFactory;
-import ui.factories.StatsWindowFactory;
+import entities.GameManager;
+import entities.IObserverNotifier;
+import entities.ObserverNotifier;
+import ui.factories.*;
+import ui.windows.Window;
 import ui.windows.layout_managers.CardLayoutManager;
```

Comment:
Consider just importing ui.windows.layout_managers.* instead of lines 9, 10, and 11. You import everything in that file anyways.

Reply:
I didn't bother fixing this as this will be overwritten soon anyways.